templates/shared_services/gitea/terraform/variables.tf

```diff
@@ -29,3 +29,18 @@ variable "acr_name" {
   type        = string
   description = "Name of Azure Container Registry"
 }
+
+variable "arm_tenant_id" {
```

Comment:
Are these needed if not running the rest command?

Reply:
Nope these won't be needed! I will update the keyVaultReferenceIdentity instead, thanks!

Reply:
@tanya-borisova can these go now?

Reply:
@marrobi yes, sorry forgot about that -- updated now